src/components/Result.tsx

```diff
@@ -38,6 +38,11 @@ const Result = () => {
               <th className={`${styles.th} ${styles.centered}`}>
                 {t('result.attempts')}
               </th>
+              {results.some((result) => result.question.referenceLink) ? (
+                <th className={styles.th}>{t('result.referenceLink')}</th>
```

Comment:
For the design we could also omit the table heading text. What do you think?

![Screenshot 2024-04-16 at 11 13 52](https://github.com/christophblessing/quiz-recap/assets/22977799/f45bc340-6689-4a40-af82-7b4d2cf2460d)

Don't forget to add an `aria-label` if adapt it like this.

Reply:
I'd keep the table heading, but if you prefer it without it, I'll remove it.

Reply:
@JuliaCasamitjana what do you think?

Reply:
Julia and I discussed it in person an we would propose the following UI:

![image](https://github.com/christophblessing/quiz-recap/assets/22977799/42eb76b2-f848-42e5-a4d6-50e3049fb5ff)

The table heading wording is "Reference" and each link in the table is "Link". What do you think?

Reply:
Yeah I think it is better. I don't think this is perfect yet, but I also don't have a better idea. 😇